script.js

```diff
@@ -1,67 +1,104 @@
+/* eslint-disable max-classes-per-file */
+
 class Books {
   constructor(title, author) {
     this.title = title;
     this.author = author;
+    this.bookShelf = [];
   }
 }
 
 const buttonAdd = document.querySelector('.button-add');
 const displayBooks = document.getElementById('books-list');
-const bookShelf = [];
 
-// Storage functions
+// Storage classes
+class Store {
+  static storeGet() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
 
-function storeGet() {
-  let books;
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
+    return books;
   }
 
-  return books;
-}
-function storeAdd(newBook) {
-  const books = storeGet();
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
-}
-function storeDelete(title) {
-  const books = storeGet();
-  const index = books.findIndex((book) => book.title === title);
-  if (index !== -1) {
-    books.splice(index, 1);
+  static storeAdd(newBook) {
+    const books = Store.storeGet();
+    books.push(newBook);
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static storeDelete(title) {
+    const books = Store.storeGet();
+    const index = books.findIndex((book) => book.title === title);
+    if (index !== -1) {
+      books.splice(index, 1);
+      localStorage.setItem('books', JSON.stringify(books));
+    }
+  }
 }
 
-const books = storeGet();
-books.forEach((book) => displayBooks.insertAdjacentHTML('afterbegin', `<div><p>${book.title}</p> <p>${book.author}</p> <button class="remove-btn">remove</button></div>`));
+// Interaction Classes
+
+class Interaction {
+  static getBooks() {
+    const books = Store.storeGet();
+    books.forEach((book) => displayBooks.insertAdjacentHTML('afterbegin',
+      `<div class="book-item">
+  <div class="book-info">
+    <p>${book.title}</p> 
+    <p> by ${book.author}</p> 
+  </div>
+  <button class="remove-btn">remove</button>
+  </div>`));
+  }
+
+  static showBooks() {
+    const author = document.getElementById('author').value;
+    const title = document.getElementById('title').value;
+    const newBook = new Books(title, author);
 
-function showBooks() {
-  const author = document.getElementById('author').value;
-  const title = document.getElementById('title').value;
-  const newBook = new Books(title, author);
+    if (!title || !author) {
+      alert('Please enter a valid title and author');
+      return;
+    }
```

Comment:
Please let's not use `window.alert()` or `window.confirm()`. 
- It’s ugly 😒
- Impossible to style
- It stops code execution and displays differently on different browsers. 

You can consider using a custom modal instead. 🤗👍